packages/contracts/contracts/interfaces/IFulfillInterpreter.sol

```diff
@@ -11,7 +11,8 @@ interface IFulfillInterpreter {
     uint256 amount,
     bytes callData,
     bytes returnData,
-    bool success
+    bool success,
+    bool isContract
```

Comment:
another nit: are event objects ordered tuples, or indexed by keys? This ordering could match TransactionFulfilled below, where `returnData` is the last thing in the tuple per se, so that's why I ask

Reply:
total nit, feel free to ignore/resolve this if you'd like. 100% your call